README.md

```diff
@@ -461,31 +465,36 @@ To register models to a different server, specify the desired server in the `mod
 <details>
 <summary><strong><u>Full configuration reference</u></strong></summary>
 
-- `uri`: string. Required, if this section is present.  
+- `registry_uri`: string. Required, if this section is present.  
 URI of the model registry server to which to register trained models.
 
+- `model_name`: string. Required.  
```

Comment:
On second thought, do we have to require this name? Can't it just be the basename of the project directory by default, if user doesn't specify it?

Reply:
Currently it is a required field. Should we try and make that optional?